synapse/models/infotech.py

```diff
@@ -775,7 +810,69 @@ def getModelDefs(self):
 
                     ('keyboard:language', ('lang:language', {}), {
                         'doc': 'The primary keyboard input language configured on the host.'}),
+
+                    ('image', ('it:software:image', {}), {
+                        'doc': 'The container image or OS image running on the host.'}),
+                )),
+
+                ('it:host:tenancy', {}, (
+
+                    ('lessor', ('it:host', {}), {
+                        'doc': 'The host which provides runtime resources to the tenant host.'}),
+
+                    ('tenant', ('it:host', {}), {
+                        'doc': 'The host which is run within the resources provided by the lessor.'}),
+
+                )),
+
+                ('it:software:image', {}, (
+
+                    ('name', ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'The name of the image.'}),
+
+                    ('type', ('it:software:image:type:taxonomy', {}), {
+                        'doc': 'The type of software image.'}),
+
+                    ('published', ('time', {}), {
+                        'doc': 'The time the image was published.'}),
+
+                    ('publisher', ('ps:contact', {}), {
+                        'doc': 'The contact information of the org or person who published the image.'}),
+
+                    ('publisher:account', ('inet:service:account', {}), {
```

Comment:
Sanity check that all props from inet:service:object interface are valid for this form (I think so, at least none seem overly pointy but I'm less familliar w/this use case).